kalite/topic_tools/__init__.py

```diff
@@ -259,6 +250,19 @@ def get_content_cache(force=False, annotate=False, language=settings.LANGUAGE_CO
         # Loop through all content items and put thumbnail urls, content urls,
         # and subtitle urls on the content dictionary, and list all languages
         # that the content is available in.
+        contents_folder = os.listdir(settings.CONTENT_ROOT)
+
+        subtitle_langs = {}
+
+        if os.path.exists(i18n.get_srt_path()):
+            for (dirpath, dirnames, filenames) in os.walk(i18n.get_srt_path()):
+                lc = dirpath.split("/")[-2]
```

Comment:
Ah, but at any rate it's only intended to work at a certain depth in the directory structure. Still, if at all possible I think it's best to avoid having garbage in variables. For instance, `dirnames` I expect is falsy if there are no subdirectories, so you could condition your code on that.